setup.py

```diff
@@ -33,6 +33,10 @@
         'Operating System :: POSIX :: Linux',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Can you take this out. I don't find it that useful given that it will probably work on newer versions too.

Reply:
personally I used these to indicate "these are the supported & tested version, don't be surprised if it doesn't work with others" instead of relying on just "probably"